src/pages/public_registration/action_accordions/DoiRequestAccordion.tsx

```diff
@@ -79,31 +72,44 @@ export const DoiRequestAccordion = ({
   const dispatch = useDispatch();
   const [isLoading, setIsLoading] = useState(LoadingState.None);
   const [messageToCurator, setMessageToCurator] = useState('');
+
   const [openRequestDoiModal, setOpenRequestDoiModal] = useState(false);
   const toggleRequestDoiModal = () => setOpenRequestDoiModal((open) => !open);
+
   const [showConfirmDialogAssignDoi, setShowConfirmDialogAssignDoi] = useState(false);
   const toggleConfirmDialogAssignDoi = () => setShowConfirmDialogAssignDoi((open) => !open);
 
   const [openReserveDoiDialog, setOpenReserveDoiDialog] = useState(false);
   const toggleReserveDoiDialog = () => setOpenReserveDoiDialog((open) => !open);
 
-  const ticketMutation = useMutation({
```

Comment:
splitter denne i to, så det blir en bare for godkjenning, og en bare for avvsing. Tror det vil forenkle litt ved at vi slipper å håndtere flere caser i samme mutation